base_delivery_carrier_files/wizard/generate_carrier_files.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2012 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, exceptions, api
+from openerp.tools.translate import _
+
+
+class DeliveryCarrierFileGenerate(models.TransientModel):
+
+    _name = 'delivery.carrier.file.generate'
+
+    @api.model
+    def _get_pickings(self):
+        context = self.env.context
+        if (context.get('active_model') == 'stock.picking' and
+                context.get('active_ids')):
+            return self.env["stock.picking"].browse(
+                context["active_ids"])
+
+    @api.multi
+    def action_generate(self):
+        """
+        Call the creation of the delivery carrier files
+        """
+        if not self.pickings:
+            raise exceptions.except_orm(_('Error'),
+                                        _('No delivery orders selected'))
+        self.pickings.generate_carrier_files(
+            auto=False, recreate=self.recreate)
+        return {'type': 'ir.actions.act_window_close'}
+
+    pickings = fields.Many2many('stock.picking',
```

Comment:
Well the convention was to use ids because it actually was ids previously, now its just a recordset right? I even remember a similar remark regarding this earlier in this MR. Do you still wish me to rename it?